build/webpack.config.js

```diff
@@ -66,11 +66,6 @@ webpackConfig.output = {
 // ------------------------------------
 webpackConfig.plugins = [
   new webpack.DefinePlugin(config.compiler_globals),
-  new webpack.ProvidePlugin({
-    $: 'jquery',
-    jQuery: 'jquery',
-    'window.jQuery': 'jquery',
-  }),
```

Comment:
We don't expose jQuery on the window up front.  We attach it to the window ourselves after `require()`ing it.  We do this so we can throw an error if the user has already loaded it on the window before we load.